node/config.go

```diff
@@ -74,6 +78,11 @@ type Config struct {
 	UseSecureGrpc                  bool
 	ReachabilityPollIntervalSec    uint64
 	DisableNodeInfoResources       bool
+	BLSRemoteSignerUrl             string
+	BLSPublicKeyHex                string
+	BLSKeyPassword                 string
+	BLSSignerTLSCertFilePath       string
+	UseBLSRemoteSigner             bool
```

Comment:
```suggestion

	BLSRemoteSignerEnabled      bool
	BLSRemoteSignerUrl             string
	BLSPublicKeyHex                string
	BLSKeyPassword                 string
	BLSSignerTLSCertFilePath       string
```

Reply:
Personally prefer to keep same naming convention for all vars, and to have the `enabled` var at the top.

Reply:
Maybe consider creating subconfigs also? The config for the node is crazy long.